### ISSUE TITLE
[Bug] Build Workflow Fails

### ISSUE THREAD

**Issue by Al4ise** [qnblackcat/uYouPlus#1206]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Device info

.

### Describe the bug

The build workflow gives this error when run. It used to not give the error a few days ago, running with the same commit.
 
<img width="1662" alt="Screenshot 2023-07-26 at 12 45 03" src="https://github.com/qnblackcat/uYouPlus/assets/39273158/35826d55-018d-46af-86e1-c3c2d8471357">

(runs on is already set to macos-11, it gives a different error on macos-latest)

### Steps to reproduce the issue

.

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

**Comment by Mukul1127** [comment]
Same

**Comment by Mukul1127** [comment]
This is caused by some of the latest commits to theos, replace your buildapp.yml with
```
# Original idea by @ISnackable. Thanks to him for handling the hardest parts!
# https://github.com/ISnackable/CercubePlus/blob/main/.github/workflows/Build.yml

name: Build and Release uYouPlus

on:
  workflow_dispatch:
    inputs:
      uyou_version:
        description: "The version of uYou"
        default: "3.0.1"
        required: true
        type: string
      decrypted_youtube_url:
        description: "The direct URL to the decrypted YouTube ipa"
        default: ""
        required: true
        type: string
      youtube_version:
        description: "The version of YouTube"
        default: ""
        required: true
        type: string
      bundle_id:
        description: "Modify the bundle ID. Not recommended"
        default: "com.google.ios.youtube"
        required: true
        type: string
      app_name:
        description: "Modify the name of the app on the Home Screen. Not recommended"
        default: "YouTube"
        required: true
        type: string
      create_release:
        description: "Create a draft release"
        default: true
        required: false
        type: boolean
  
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true

jobs:
  build:
    name: Build uYouPlus
    runs-on: macos-latest
    permissions:
      contents: write

    steps:
      - name: Checkout Main
        uses: actions/checkout@v3.5.3
        with:
          path: main
          submodules: recursive

      - name: Install Dependencies
        run: brew install ldid dpkg make

      - name: Set PATH environment variable
        run: echo "$(brew --prefix make)/libexec/gnubin" >> $GITHUB_PATH 

      - name: Setup Theos
        uses: actions/checkout@v3.5.3
        with:
          repository: theos/theos
          ref: 3da31488281ecf4394d10302d2629607f4a1aa07
          path: theos
          submodules: recursive
      
      - name: Caching
        id: SDK
        uses: actions/cache@v3.3.1
        env:
          cache-name: iOS-15.5-SDK
        with:
          path: theos/sdks/
          key: ${{ env.cache-name }}
      
      - name: Download iOS 15.5 SDK
        if: steps.SDK.outputs.cache-hit != 'true'
        run: |
          svn checkout -q https://github.com/chrisharper22/sdks/trunk/iPhoneOS15.5.sdk
          mv *.sdk $THEOS/sdks
        env:
          THEOS: ${{ github.workspace }}/theos

      - name: Setup Theos Jailed
        uses: actions/checkout@v3.5.3
        with:
          repository: qnblackcat/theos-jailed
          ref: master
          path: theos-jailed
          submodules: recursive

      - name: Install Theos Jailed
        run: |
          ./theos-jailed/install
        env:
          THEOS: ${{ github.workspace }}/theos

      - name: Download YouTube iPA
        run: |
          wget "$YOUTUBE_URL" --no-verbose -O main/YouTube.ipa && echo -e "==> \033[1mYouTube v${{ inputs.youtube_version }} downloaded! \033[0m"

        env:
          THEOS: ${{ github.workspace }}/theos
          YOUTUBE_VERSION: ${{ inputs.youtube_version }}
          YOUTUBE_URL: ${{ inputs.decrypted_youtube_url }}

      - name: Fix compiling & Build Package
        id: build_package
        run: |
          cd ${{ github.workspace }}/main
          sed -i '' "27s#.*#BUNDLE_ID = ${{ env.BUNDLE_ID }}#g" Makefile
          sed -i '' "26s#.*#DISPLAY_NAME = ${{ env.APP_NAME }}#g" Makefile          
          make package IPA=YouTube.ipa FINALPACKAGE=1
          (mv "packages/$(ls -t packages | head -n1)" "packages/uYouPlus_${{ env.YOUTUBE_VERSION  }}_${{ env.UYOU_VERSION }}.ipa")
          echo "package=$(ls -t packages | head -n1)" >>$GITHUB_OUTPUT
          echo -e "==> \033[1mSHASUM256: $(shasum -a 256 packages/*.ipa | cut -f1 -d' ')\033[0m"
          echo -e "==> \033[1mBundle ID: ${{ env.BUNDLE_ID }}\033[0m"

        env:
          THEOS: ${{ github.workspace }}/theos
          UYOU_VERSION: ${{ inputs.uyou_version }}
          YOUTUBE_VERSION: ${{ inputs.youtube_version }}
          BUNDLE_ID: ${{ inputs.bundle_id }}
          APP_NAME: ${{ inputs.app_name }}

      - name: Upload Artifact
        uses: actions/upload-artifact@v3.1.1
        env:
          UYOU_VERSION: ${{ inputs.uyou_version }}
          YOUTUBE_VERSION: ${{ inputs.youtube_version }}
        with:
          name: uYouPlus_${{ env.YOUTUBE_VERSION }}_${{ env.UYOU_VERSION }}
          path: ${{ github.workspace }}/main/packages/${{ steps.build_package.outputs.package }}
          if-no-files-found: error

      - name: Create Release
        id: create_release
        uses: softprops/action-gh-release@v0.1.15
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          UYOU_VERSION: ${{ inputs.uyou_version }}
          YOUTUBE_VERSION: ${{ inputs.youtube_version }}
          DRAFT: ${{ inputs.create_release }}
        with:
          tag_name: v${{ env.YOUTUBE_VERSION }}-${{ env.UYOU_VERSION }}-(${{ github.run_number }})
          name: v${{ env.YOUTUBE_VERSION }}-${{ env.UYOU_VERSION }}-(${{ github.run_number }})
          files: main/packages/*.ipa
          draft: ${{ env.DRAFT }}
```
This buildapp is also fixed for macos-latest and uses uYou 3.0.1 by default

**Comment by asdfzxcvbn** [comment]
> This buildapp is also fixed for macos-latest and uses uYou 3.0.1 by default

it actually didnt work for me using macos-latest, but it worked on macos-11! thanks!

**Comment by Mukul1127** [comment]
> > This buildapp is also fixed for macos-latest and uses uYou 3.0.1 by default
> 
> it actually didnt work for me using macos-latest, but it worked on macos-11! thanks!

I think this is a issue with the code as uYouPlusExtra builds just fine on macos-latest

**Comment by UnoCap** [comment]
Apply this commit: https://github.com/arichorn/uYouPlusExtra/commit/0c20ac5305265377d512a580525cadcc4d1b8304
